helmfile/values/opendistro-es.yaml.gotmpl

```diff
@@ -450,26 +418,37 @@ curator:
     secret: opendistro-es-curator-user
 
   retention:
-    other_gb: {{ .Values.elasticsearch.retention.otherSize }}
-    other_days: {{ .Values.elasticsearch.retention.otherAge }}
-    kubernetes_gb: {{ .Values.elasticsearch.retention.kubernetesSize }}
-    kubernetes_days: {{ .Values.elasticsearch.retention.kubernetesAge }}
-    kubeaudit_gb: {{ .Values.elasticsearch.retention.kubeAuditSize }}
-    kubeaudit_days: {{ .Values.elasticsearch.retention.kubeAuditAge }}
-
+    other_gb: {{ .Values.elasticsearch.curator.retention.otherSize }}
+    other_days: {{ .Values.elasticsearch.curator.retention.otherAge }}
+    kubernetes_gb: {{ .Values.elasticsearch.curator.retention.kubernetesSize }}
+    kubernetes_days: {{ .Values.elasticsearch.curator.retention.kubernetesAge }}
+    kubeaudit_gb: {{ .Values.elasticsearch.curator.retention.kubeAuditSize }}
+    kubeaudit_days: {{ .Values.elasticsearch.curator.retention.kubeAuditAge }}
+    postgresql_gb: {{ .Values | get "elasticsearch.curator.retention.postgresqlSize" nil }}
+    postgresql_days: {{ .Values | get "elasticsearch.curator.retention.postgresqlAge" nil }}
```

Comment:
It would be nice to keep the units in the helmfile values so we don't have to go here to check them. For example `otherSizeGb` and `otherAgeDays` or something like that. What do you think?

Reply:
So you mean like this?
```
# sc-config
curator.retention.otherAgeDays: 10

# opendsitro-es.yaml.gotmpl
other_days: {{ .Values.elasticsearch.curator.retention.otherAgeDays }}d
```

Reply:
Yes exactly! :slightly_smiling_face: I think it would be very nice to have the unit there explicitly.

Reply:
Done